addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -691,6 +691,16 @@ static func get_built_in_blocks(_class_name: String) -> Array[Block]:
 static func _get_input_blocks() -> Array[Block]:
 	var block_list: Array[Block]
 
+	var editor_input_actions: Dictionary = {}
+	var editor_input_action_deadzones: Dictionary = {}
+	if Engine.is_editor_hint():
+		var actions := InputMap.get_actions()
+		for action in actions:
+			if action.begins_with("spatial_editor"):
```

Comment:
@Cafezinhu are there other actions worth re-adding other than the ones beginning with "spatial_editor"?